test/api/vim/buffer.vader

```diff
@@ -3,5 +3,7 @@ Execute ( SpaceVim api: vim#buffer ):
   let buffer = SpaceVim#api#import('vim#buffer')
   normal! g:
```

Comment:
here should be `call feedkeys('q:', 'n')`.  and issue is opened here : https://github.com/junegunn/vader.vim/issues/129